pinot-common/src/main/java/org/apache/pinot/common/metadata/ZKMetadataProvider.java

```diff
@@ -112,14 +112,14 @@ public static String constructPropertyStorePathForInstancePartitions(String inst
     return StringUtil.join("/", PROPERTYSTORE_INSTANCE_PARTITIONS_PREFIX, instancePartitionsName);
   }
 
-  public static String constructPropertyStorePathForControllerJob() {
-    return StringUtil.join("/", PROPERTYSTORE_CONTROLLER_JOBS_PREFIX);
-  }
-
   public static String constructPropertyStorePathForResource(String resourceName) {
     return StringUtil.join("/", PROPERTYSTORE_SEGMENTS_PREFIX, resourceName);
   }
 
+  public static String constructPropertyStorePathForControllerJobResource(String resourceName) {
```

Comment:
Are we going to create one ZNode per table?

Reply:
Reading the following code, we are actually passing the job type instead of the resource name (resource name is the table name with type, e.g. `myTable_OFFLINE`)

Reply:
We are creating one Znode per jobType actually. We have 3 jobTypes (reload, force_commit, reload_all).